package-lock.json

```diff
@@ -28242,7 +28268,7 @@
 		},
 		"packages/pages-shared": {
 			"name": "@cloudflare/pages-shared",
-			"version": "0.0.14-prerelease-1",
+			"version": "0.0.13",
```

Comment:
What's this change about?

Reply:
hmmm...I don't think that should be there. @GregBrimble is this smth we should address in a separate PR?

Reply:
oh! ok then...changing back